### DISCUSSION TITLE
Build error:  invalid target release: 11

### DISCUSSION THREAD

**Discussion by yangmeng66** [Apicurio/apicurio-registry#2292] [Q&A]
hello!  when use  ./mvnw clean install -DskipTests

![image](https://user-images.githubusercontent.com/51857567/155838958-79bcbad4-e7be-4f39-b27a-e16acb3a1597.png)

![image](https://user-images.githubusercontent.com/51857567/155838968-c31ac5de-2ffd-4615-987a-42a79c7c5e8c.png)

**Comment by paoloantinori** [comment]
Can you please re-run with `-X` and attach full logs?
